python/pyspark/sql/__init__.py

```diff
@@ -72,4 +73,5 @@
     "DataFrameWriter",
     "DataFrameWriterV2",
     "PandasCogroupedOps",
+    "is_remote",
```

Comment:
Can we expose this as `pyspark.sql.utils.is_remote`?

Reply:
I think we should just expose it under `pyspark.sql` because APIs are usually exposed under that namespace. I think you can use it as:

```python
is_remote = getattr(pyspark.sql, "is_remote", getattr(pyspark.sql.utils, "is_remote", None))()
```